tests/testimages/genref

```diff
@@ -90,6 +90,9 @@ $toktx --nowarn --t2 --test --input_swizzle 0rr1 --target_type RGBA cyan_rgba_re
 $toktx --nowarn --t2 --test --input_swizzle 0rr1 --target_type RGBA --bcmp cyan_rgb_reference_basis.ktx2 ../srcimages/level0.ppm
 $toktx --nowarn --t2 --test --input_swizzle 0rr1 --target_type RGBA --uastc -- cyan_rgb_reference_uastc.ktx2 ../srcimages/level0.ppm
 
+$toktx --nowarn --t2 --test --assign_oetf linear --normal_mode --encode uastc uastc_Iron_Bars_001_normal.ktx2 ../srcimages/Iron_Bars/Iron_Bars_001_normal_unnormalized.png
```

Comment:
We need `--normalize` when generating all 4 of the comparison files for the normal tests.

Reply:
fixed in https://github.com/KhronosGroup/KTX-Software/pull/493/commits/a3f67c459290e2c249226cdd685231f7452d0dba